panel/io/admin.py

```diff
@@ -75,6 +73,44 @@ def emit(self, record):
         self._data.param.trigger('data')
 
 
+class _LogTabulator(Tabulator):
+
+    def __init__(self, **params):
+        params["value"] = self._create_frame()
+        params["theme"] = "midnight"
+        params["layout"] = "fit_data_stretch"
+        params["show_index"] = False
+        params["sorters"] = [{'field': 'datetime', 'dir': 'dsc'}]
+        params["disabled"] = True
+        params["pagination"] = "local"
+        params["page_size"] = 18
```

Comment:
Can we maybe move this onto a class parameter as a single dictionary so someone could override these defaults? Alternatively I suppose the correct thing to do would be to override the actual parameters with new defaults.

Reply:
Is there a best practice for overriding the defaults?